### PR TITLE
[Android Upload Issue]

### PR DESCRIPTION
**Pull request by cackrogt**
[Issue] Not able to Upload large files, causes crash

[Solution] Starting soln. Add the upload file part in a thread.

### CODE REVIEW

File: .idea/gradle.xml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="GradleMigrationSettings" migrationVersion="1" />
```
Comment (Aceee-dev) [inline .idea/gradle.xml:3]: Is this change required for the purpose of this patch or is it some local change. If not, can we remove it from this patch.

**Review by Aceee-dev — APPROVED** [review]
Please check nit comment and address it if required.

File: .idea/gradle.xml (hunk 1 above)
Comment (cackrogt) [reply .idea/gradle.xml:3]: Yeah, will check if we can remove, if it isn't a lot of trouble. anyways, this pr will be gone, i will merge this change with your branch, and the result will be merged with the main branch. Anyways this is partial success. Will have to work on the server related logic now
